src/components/shared/transactionsTable/transactionRow.js

```diff
@@ -18,7 +18,7 @@ const TransactionRow = ({
   data, className, t, currentBlockHeight,
 }) => (
   <DialogLink
-    className={`${grid.row} ${className}`}
+    className={`${grid.row} ${className} transactions-row`}
```

Comment:
Same as `transactions-results`

Reply:
@isalga I don't think so. `transaction-results` refers to the table container and as a result it can't be used directly to count the number of rows in the table, that's why I needed the `transactions-row`